# Set your detection topic here
detection_topic: "/object_detection/detections"

# Set your output topic here
tracker_topic: "/object_tracker/tracks"

# Threshold for Costs between detections and tracks
cost_threhold: 15
